plugins/default/src/Ide/Plugin/Pragmas.hs

```diff
@@ -61,14 +65,14 @@ codeActionProvider _ state _plId docId _ (J.CodeActionContext (J.List diags) _mo
     let dflags = ms_hspp_opts . pm_mod_summary <$> pm
         -- Get all potential Pragmas for all diagnostics.
         pragmas = nubOrd $ concatMap (\d -> genPragma dflags (d ^. J.message)) diags
-    cmds <- mapM mkCodeAction pragmas
+    cmds <- mapM (mkCodeAction pm) pragmas
```

Comment:
It's better to handle `Maybe`ness before here. If not, it will generate a bunch of `Nothing` edits...